src/components/pages/HomePage.jsx

```diff
@@ -15,13 +15,18 @@ const HomePage = () => {
 
 
   return (
-    <>
+    <div className="flex flex-col min-h-screen">
+   <main className="flex-grow p-4">
       <h1 className="text-3xl font-spartan font-bold">HOME PAGE</h1>
       <button onClick={search}>Search classes</button>
       <button onClick={login}>Log In</button>
       <button className='bg-red hover:bg-pureWhite hover:text-red hover:border-2 hover:border-red text-white font-spartan font-semibold text-lg py-1 px-7 rounded-lg' 
               onClick={register}>Join</button>
-    </>
+        </main>
+        <Footer />
```

Comment:
Please remove the footer component from the main page. It's a common component, so it should be in Approutes

Reply:
I believe that the tag main should also be in Approutes